ui/components/component-library/popover-header/popover-header.test.tsx

```diff
@@ -6,32 +6,39 @@ import { PopoverHeader } from './popover-header';
 describe('PopoverHeader', () => {
   it('should render PopoverHeader correctly', () => {
     const { getByTestId, container } = render(
-      <PopoverHeader data-testid="popover-header">
-        Popover Header
-      </PopoverHeader>,
+      <PopoverHeader data-testid="popover-header">PopoverHeader</PopoverHeader>,
     );
     expect(getByTestId('popover-header')).toHaveClass('mm-popover-header');
     expect(container).toMatchSnapshot();
   });
 
-  it('should render popover header title', () => {
+  it('should render popover header children as a string', () => {
     const { getByText } = render(
       <PopoverHeader data-testid="popover-header">
-        Popover Header Test
+        PopoverHeader test
+      </PopoverHeader>,
+    );
+    expect(getByText('PopoverHeader test')).toBeDefined();
+  });
+
+  it('should render popover header children as a node', () => {
+    const { getByText, getByTestId } = render(
+      <PopoverHeader data-testid="popover-header">
+        <div data-testid="div">PopoverHeader test</div>
       </PopoverHeader>,
     );
-    expect(getByText('Popover Header Test')).toBeDefined();
+    expect(getByText('PopoverHeader test')).toBeDefined();
+    expect(getByTestId('div')).toBeDefined();
```

Comment:
Increasing test coverage for `PopoverHeader`

### Before
<img width="792" alt="Screenshot 2023-04-25 at 8 52 54 AM" src="https://user-images.githubusercontent.com/8112138/234164547-54acd735-81cd-45d6-88d1-96f6bf9c2157.png">

### After

<img width="746" alt="Screenshot 2023-04-25 at 8 54 07 AM" src="https://user-images.githubusercontent.com/8112138/234164560-841b77c5-323c-4d7e-b313-3dc1b8004d6e.png">